app/src/main/java/com/google/droidjump/MainActivity.java

```diff
@@ -49,9 +50,10 @@
     // Request code used to invoke sign in user interactions.
     private static final int RC_SIGN_IN = 9001;
     private static final String TAG = "MainActivity";
-    private String playerId;
+    private Player player = null;
```

Comment:
nit: move all variable initializations to the onCreate method besides final ones.